package.json

```diff
@@ -3,19 +3,24 @@
   "version": "0.0.1",
   "description": "placeholder-app-description",
   "scripts": {
-    "postinstall": "npm run build-app",
-    "build-app": "cd app && npm install && cd ..",
+    "postinstall": "yarn compile && yarn copy-artifacts && yarn build-app",
```

Comment:
Are we making `yarn` mandatory for the boilerplate?

Reply:
No, you are right this should not be updated